ref/addons.md

```diff
@@ -22,15 +21,16 @@ To create the addon, you must provide `amc` with either of the following:
 
 The following table lists the valid keys in an addon manifest:
 
-| Name                  | Type         | Description                                                                                                                                                                                                                   | Allowed values                        |
-|-----------------------|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
-| `name`                | string       | Name of the addon. Can be used to reference the addon.                                                                                       | All characters except for the following: `< > : " / \ \| ? *`, as well as space. |
-| `description`         | string       | Description of the addon.                                                                                                                                                                                                     |                                       |
-| `provides`            | object       | Additional capabilities that this addon provides to the container. See individual items for details.                                                                                                                          |                                       |
+| Name                  | Type         | Description    | Allowed values  |
+|-----------------------|--------------|----------------|-----------------|
+| `name`                | string       | Name of the addon. Can be used to reference the addon.| All characters except for the following: `< > : " / \ \| ? *`, as well as space. |
+| `description`         | string       | Description of the addon.   |        |
+| `provides`            | object       | Additional capabilities that this addon provides to the container. See individual items for details.          |             |
```

Comment:
Shouldn't you also add something in the 'Allowed values' column? I guess these two can contain any character (ASCII? UTF-8?), but we should maybe check.

Reply:
yes, I will check this and add relevant information.

Reply:
Dropped the `provides` as discussed in SU.